db/seeds.rb

```diff
@@ -8,6 +8,7 @@
                     fetch(Rails.env, {})
 
 service_providers.each do |issuer, config|
+  next if Figaro.env.chef_env == 'prod' && config['live'] != 'true'
```

Comment:
No I think the question was not about how to check which env, but if the check should be inside or outside the loop, since the env doesn't change per loop iteration